diff --git a/xwalk/runtime/browser/ui/native_app_window_views.cc b/xwalk/runtime/browser/ui/native_app_window_views.cc
index 1cb3540..221273f 100644
--- a/xwalk/runtime/browser/ui/native_app_window_views.cc
+++ b/xwalk/runtime/browser/ui/native_app_window_views.cc
@@ -14,9 +14,7 @@
 #include "xwalk/runtime/browser/ui/xwalk_views_delegate.h"
 #include "xwalk/runtime/common/xwalk_notification_types.h"
 
-#if defined(OS_WIN)
 #include "ui/views/window/native_frame_view.h"
-#endif
 
 #if defined(OS_TIZEN)
 #include "xwalk/runtime/browser/ui/native_app_window_tizen.h"
@@ -235,12 +233,10 @@ bool NativeAppWindowViews::CanMinimize() const {
   return true;
 }
 
-#if defined(OS_WIN)
 views::NonClientFrameView* NativeAppWindowViews::CreateNonClientFrameView(
     views::Widget* widget) {
   return new views::NativeFrameView(widget);
 }
-#endif
 
 ////////////////////////////////////////////////////////////
 // views::View implementation
diff --git a/xwalk/runtime/browser/ui/native_app_window_views.h b/xwalk/runtime/browser/ui/native_app_window_views.h
index 413bd0b..4a36036 100644
--- a/xwalk/runtime/browser/ui/native_app_window_views.h
+++ b/xwalk/runtime/browser/ui/native_app_window_views.h
@@ -79,10 +79,8 @@ class NativeAppWindowViews : public NativeAppWindow,
   bool CanResize() const override;
   bool CanMaximize() const override;
   bool CanMinimize() const override;
-#if defined(OS_WIN)
   views::NonClientFrameView* CreateNonClientFrameView(
       views::Widget* widget) override;
-#endif
   // views::View implementation.
   void ChildPreferredSizeChanged(views::View* child) override;
   void OnFocus() override;
